Formula/t/typstfmt.rb

```diff
@@ -0,0 +1,21 @@
+class Typstfmt < Formula
+  desc "Formatter for typst"
+  homepage "https://github.com/astrale-sharp/typstfmt"
+  url "https://github.com/astrale-sharp/typstfmt/archive/refs/tags/0.2.7.tar.gz"
+  sha256 "17db7ddb83a8e2e43b3b34f19dcc8cbd3585aea34e05a5689e6f5beda27ac523"
+  license "MIT"
```

Comment:
- there is some license issue, https://github.com/astrale-sharp/typstfmt/issues/153

Reply:
Should we change it to `MIT OR Apache-2.0` here?

Reply:
We should be sure what the license is before doing either.

Reply:
updated it to `one_of: ["MIT", "Apache-2.0"]`